packages/react/src/components/table/table.tsx

```diff
@@ -37,6 +37,7 @@ type CustomColumn<T extends object> = Column<T> & UseSortByColumnOptions<T> & {
     textAlign?: string,
     className?: string,
     sticky?: boolean,
+    colSpan?: string,
```

Comment:
That only applies to the footer so it should be called something like `footerColSpan` or `footerSpan`.

Reply:
Also, wouldn't that make more sense if it was a number?

Reply:
Number would align with colSpan in React.

```
React.TdHTMLAttributes<HTMLTableDataCellElement>.colSpan?: number | undefined
```

Reply:
done